app/src/main/java/com/github/se/polyfit/data/user/User.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.se.polyfit.model.data
+
+import android.net.Uri
+
+data class User(
+    val id: String?,
+    val displayName: String?,
+    val familyName: String?,
+    val getGivenName: String?,
+    val email: String?,
+    val photoURL: Uri?
+) {
+  companion object {
+    fun resetCurrentUser() {
```

Comment:
Instead of setting the pointer to null, it could be really useful to have a method that resets the fields to default values. Did you add this method in anticipation of implementing a logout feature?